docs/docs/scanner/vulnerability.md

```diff
@@ -98,6 +99,12 @@ See [here](../coverage/language/index.md#supported-languages) for the supported
 
 [^1]: Intentional delay between vulnerability disclosure and registration in the DB
 
+## Kubernetes components
```

Comment:
```suggestion
## Kubernetes components (control plane, node and addons)
```

Reply:
changed in https://github.com/aquasecurity/trivy/pull/5268/commits/7c98227bac45948cfcd5d4799c1d010e7d3d6f33